src/utilData.js

```diff
@@ -0,0 +1,4 @@
+export const YOUTUBE_LINK =
+  "https://www.googleapis.com/youtube/v3/videos?part=snippet&part=statistics&chart=mostPopular&maxResults=50";
+
+export const API_KEY = "AIzaSyBaKOviqO4JBR2O4f5MSze0v-DijMGSEUw";
```

Comment:
Your API key is exposed you don't want to run out of the limit.
use dotenv to store the API key